### PR TITLE
(chore) Update COC to OpenJS

### PR DESCRIPTION
**Pull request by chrisgervang**
Based this on Electron's, which is another OpenJS project.

### CODE REVIEW

File: contributing/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,75 +1,135 @@
-# Contributor Covenant Code of Conduct
+# Code of Conduct
 
-## Our Pledge
+As a member project of the OpenJS Foundation, kepler.gl uses [Contributor Covenant v2.0](https://contributor-covenant.org/version/2/0/code_of_conduct) as their code of conduct. The full text is included [below](#contributor-covenant-code-of-conduct) in English, and translations are available from the Contributor Covenant organisation:
```
Comment (ibgreen) [inline contributing/CODE_OF_CONDUCT.md:3]:
```suggestion
As a member project of the OpenJS Foundation, kepler.gl uses [Contributor Covenant v2.0](https://contributor-covenant.org/version/2/0/code_of_conduct) as our code of conduct. The full text is included [below](#contributor-covenant-code-of-conduct) in English, and translations are available from the Contributor Covenant organisation:
```

File: contributing/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,75 +1,135 @@
...
-Examples of behavior that contributes to creating a positive environment
-include:
+### Our Pledge
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+We as members, contributors, and leaders pledge to make participation in our
```
Comment (ibgreen) [inline contributing/CODE_OF_CONDUCT.md:12]: Do we need to add all this text when we are already referencing a master doc?

**Review by ibgreen — APPROVED** [review]

**Comment by ibgreen** [comment]
@chrisgervang Can you `--signoff` your commit to make CI pass?

**Comment by chrisgervang** [comment]
Actually, I'd like to take this in a different direction following some more discussion with OpenJS. They think we can just use a short CoC and link to theres.

**Pull request merged**